Tests/TestPlans/Apollo-CodegenTestPlan.xctestplan

```diff
@@ -13,6 +13,7 @@
   },
   "testTargets" : [
     {
+      "parallelizable" : true,
```

Comment:
I'm not certain what level of service we pay for in CircleCI. This could work against us if the shared machines we run CI on become saturated with other jobs too. Appears to give a nice improvement but worth keeping an eye on and remembering if we see odd test behaviour. Nice find!